src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.cmd

```diff
@@ -56,6 +56,24 @@ if not [%NET46_PUBLISH_ERROR_LEVEL%]==[0] (
 	exit /b %NET46_PUBLISH_ERROR_LEVEL%
 )
 
+set ILASM_PROJECTJSON=%BUILDTOOLS_PACKAGE_DIR%\ilasm\project.json
```

Comment:
Perhaps I'm missing something but why does it need to be different? Why can it not be along side everything else? Is it because of the updated CoreCLR package? Is that actually needed for this to function?